Therapeut Vechter/Assets/Scripts/LevelSelect.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+[CreateAssetMenu(menuName = "ScriptableObjects/levelObject")]
+
+public class LevelSelect : ScriptableObject
+{
+    public string LevelName;
+    public int LevelNumber;
+    public int HeightValue;

```

Comment:
Would be worth it to implement ranges to prevent spawning the levels off screen.